### DISCUSSION TITLE
A section on cryptography-based functions

### DISCUSSION THREAD

**Discussion by MaxCorti** [josdejong/mathjs#3169] [Ideas]
A criptography-based package with direct implementations of theories (Fermat, Euler, etc.) and equivalencies for RSA algorithms and other use cases could be an interesting and helpful feature for the library.

**Comment by josdejong** [comment]
Thanks for your suggestion. I'm not sure if this has a broad use case cryptography for mathjs. It is also possible to implement such features as an extension for mathjs.
